service-commands/scripts/init-repos.sh

```diff
@@ -16,7 +16,6 @@ fi
 cd $PROTOCOL_DIR/
 cd service-commands/
 npm install
-npm install lodash # fry_kek
```

Comment:
I would not remove any of these `npm install` lines. We did multiple tests and found when these were added, in this specific order, it usually allowed our building remote-dev to succeed as expected.